docs/en/operation/User-CLI.md

```diff
@@ -882,6 +889,8 @@ $ ./bin/alluxio fs distributedLoad /data/today --locality-file /tmp/localityfile
 $ ./bin/alluxio fs distributedLoad /data/today --excluded-locality ROCK1,ROCK2
 # Include all workers except which's locality belong to the localities in the excluded locality file
 $ ./bin/alluxio fs distributedLoad /data/today --excluded-locality-file /tmp/localityfile-exclude
+# Turn on async submission mode. Run this command to get JOB_CONTROL_ID, then use `getCmdStatus` to check command detailed status.
+$ ./bin/alluxio fs distributedLoad /data/today --async
```

Comment:
don't use backticks inside the code block because they will actually show up as backticks instead of code font. but more importantly, seems easier to show how to use getCmdStatus instead of describing it
```suggestion
# Turn on async submission mode. The command will output a JOB_CONTROL_ID
$ ./bin/alluxio fs distributedLoad /data/today --async
# Get the status of the running job
$ ./bin/alluxio fs getCmdStatus $JOB_CONTROL_ID
```

also this section describes many of the optional flags that may not usually be used, but async mode should probably be promoted better. so i would add this section at the end of the first console section, immediately before `Or you can include some workers ...`

Reply:
Done, good point, thanks!